spa/src/pages/ConfigSteps/OrgsContainer/index.tsx

```diff
@@ -103,66 +81,68 @@ const OrganizationsList = ({
 		}
 	};
 	return (
-		<>
-			<div css={orgsWrapperStyle} ref={containerRef}>
+		<Scrollbars
+			style={{
+			maxHeight: 250,
```

Comment:
Nit: can we avoid using magic numbers here and opt for constants? Will just make the code a little more readable.

Reply:
Done